includes/site-health/audit-autoloaded-options/helper.php

```diff
@@ -144,13 +144,70 @@ function perflab_aao_get_autoloaded_options_table() {
 	$autoload_summary = perflab_aao_query_autoloaded_options();
 
 	$html_table = sprintf(
-		'<table class="widefat striped"><thead><tr><th scope="col">%s</th><th scope="col">%s</th></tr></thead><tbody>',
-		__( 'Option Name', 'performance-lab' ),
-		__( 'Size', 'performance-lab' )
+		'<table class="widefat striped"><thead><tr><th scope="col">%s</th><th scope="col">%s</th><th scope="col">%s</th></tr></thead><tbody>',
+		esc_html__( 'Option Name', 'performance-lab' ),
+		esc_html__( 'Size', 'performance-lab' ),
+		esc_html__( 'Action', 'performance-lab' )
 	);
 
+	$nonce = wp_create_nonce( 'perflab_aao_update_autoload' );
 	foreach ( $autoload_summary as $value ) {
-		$html_table .= sprintf( '<tr><td>%s</td><td>%s</td></tr>', $value->option_name, size_format( $value->option_value_length, 2 ) );
+		$url            = esc_url_raw(
+			add_query_arg(
+				array(
+					'action'      => 'perflab_aao_update_autoload',
+					'_wpnonce'    => $nonce,
+					'option_name' => esc_attr( $value->option_name ),
+					'autoload'    => 'no',
+					'value'       => esc_attr( $value->option_value_length ),
+				),
+				admin_url( 'site-health.php' )
+			)
+		);
+		$disable_button = sprintf( '<a class="button" href="%s">%s</a>', esc_url( $url ), esc_html__( 'Disable Autoload', 'performance-lab' ) );
+		$html_table    .= sprintf( '<tr><td>%s</td><td>%s</td><td>%s</td></tr>', esc_html( $value->option_name ), size_format( $value->option_value_length, 2 ), $disable_button );
+	}
+	$html_table .= '</tbody></table>';
+
+	return $html_table;
+}
+
+/**
+ * Gets disabled autoload options table.
+ *
+ * @since n.e.x.t
+ *
+ * @return string HTML formatted table.
+ */
+function perflab_aao_get_disabled_autoloaded_options_table() {
+	$perflab_aao_disabled_options = get_option( 'perflab_aao_modified_options', array() );
+
+	if ( empty( $perflab_aao_disabled_options ) ) {
+		return '';
+	}
+
+	$html_table = sprintf(
+		'<br><br><table class="widefat striped"><thead><tr><th scope="col">%s</th><th scope="col">%s</th><th scope="col">%s</th></tr></thead><tbody>',
+		esc_html__( 'Option Name', 'performance-lab' ),
+		esc_html__( 'Size', 'performance-lab' ),
+		esc_html__( 'Action', 'performance-lab' )
+	);
+
+	$nonce = wp_create_nonce( 'perflab_aao_update_autoload' );
+	foreach ( $perflab_aao_disabled_options as $option_name => $value ) {
+		$url            = esc_url_raw(
+			add_query_arg(
+				array(
+					'action'      => 'perflab_aao_update_autoload',
+					'_wpnonce'    => $nonce,
+					'option_name' => esc_attr( $option_name ),
```

Comment:
No escaping should be done here. It will be handled when output below.

```suggestion
					'option_name' => $option_name,
```